test/fuzz/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2024 Intel Corporation
+# Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+# See LICENSE.TXT
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+function(add_fuzz_test name label)
+    add_test(
+        NAME ${name}
+        COMMAND fuzz_test-base ${ARGN} -verbosity=1
+        WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
+    set_tests_properties(${name} PROPERTIES LABELS ${label})
+endfunction()
+
+set(TEST_TARGET_NAME fuzz_test-base)
```

Comment:
nit: Simply `fuzztest` would make it as we are not planning to add more binaries now.

Reply:
Done